resources/metadata/helmExecute.yaml

```diff
@@ -56,94 +55,47 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
-      - name: containerRegistryPassword
-        description: Password for container registry access - typically provided by the CI/CD environment.
+      - name: targetChartRepositoryURL
+        description: "URL of the target repository where the compiled helm .tgz archive shall be uploaded - typically provided by the CI/CD environment."
         type: string
         scope:
           - PARAMETERS
           - STAGES
           - STEPS
-        secret: true
         resourceRef:
-          - name: dockerCredentialsId
-            type: secret
-            param: password
           - name: commonPipelineEnvironment
-            param: custom/repositoryPassword
-      - name: containerImageName
-        aliases:
-          - name: dockerImageName
+            param: custom/repositoryUrl
+      - name: targetChartRepositoryName
         type: string
-        description: Name of the container which will be built - will be used together with `containerImageTag` instead of parameter `containerImage`
+        description: set the chart repository
+        default: "stable"
```

Comment:
do we need this default `stable` ?

Reply:
if we don't have value of targetRepositoryName in config.yaml, command execution "helm repo add" will fail, since "helm repo add" command has 2 required parameters name and url of chart

Reply:
as discussed use `mandatoryIf` condition instead:

```
        mandatoryIf:
          - name: secretStore
            value: ado
```

Reply:
done